### PR TITLE
WIP: Create an IBC knowledge center with resources, videos, tutorial..etc to help serve as a beginning for onboarding to IBC working group

### PR DESCRIPTION
**Pull request by Michelle-L**

### CODE REVIEW

**Comment by cwgoes** [comment]
As it relates to the working group, should this document go in the `ecosystem` folder?

**Comment by Michelle-L** [comment]
I think that maybe having a knowledge section within the ecosystem folder makes sense.  It would serve as an easy way to get all the information needed to be a contributor and involved in the ecosystem. How can I help?

**Comment by cwgoes** [comment]
I was suggesting that we move this Markdown document to that folder.

File: IBC knowledge center/Readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+**The Inter-Blockchain Communication Knowledge Center**
```
Comment (fedekunze) [inline IBC knowledge center/Readme.md]:
```suggestion
# The Inter-Blockchain Communication Knowledge Center
```

File: IBC knowledge center/Readme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+**Getting Started**
```
Comment (fedekunze) [inline IBC knowledge center/Readme.md]:
```suggestion
## Getting Started
```

File: IBC knowledge center/Readme.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Chain Reaction #1: [Launching an Interoperability Protocol](https://www.youtube.com/watch?v=pWsy0BD3DnY) (Zaki Manian, Tendermint and Dean Tribble, Agoric)
+- Chain Reaction #2: [Getting IBC hackathon ready!](https://www.youtube.com/watch?v=ZChn6E8Nz_s) (Zaki Manian, Tendermint and Jack Zampolin, Tendermint)
+- Chain Reaction #3: [First Cosmos SDK IBC demo introduction (November 2019)](https://www.youtube.com/watch?v=DeMhlEck3E0)
+
+Demos and tutorials
+
+- Link
+
+**General Information**
```
Comment (fedekunze) [inline IBC knowledge center/Readme.md]:
```suggestion
## General Information
```

File: IBC knowledge center/Readme.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Introduction to IBC video series with Chris Goes, Tendermint and founding member of the IBC working group and Dean Tribble, CEO of Agoric and founding member of the IBC working group. 
+
+- What is IBC?
+- How was IBC designed?
+- How will the IBC protocol develop?
+- How can you use IBC
+
+**Presentations****
```
Comment (fedekunze) [inline IBC knowledge center/Readme.md]:
```suggestion
## Presentations
```

**Comment by cwgoes** [comment]
@Michelle-L Do you still think this makes sense? It will likely be superseded by the IBC website.

**Comment by Michelle-L** [comment]
@cwgoes I agree, actually do we have an update on the IBC website. @nassdonald

**Comment by nassdonald** [comment]
This is all wonderful content, and we definitely need a place to collect these resources.
It's on the Design Team's roadmap to create a dedicated standalone website for IBC before launch (🤞 ), as well as a high-level overview page on the Cosmos site (which will probably come first), so I'll keep y'all updated on when that's likely to happen.

**Pull request closed without merging**